src/components/ImportRecordsFlow/stepOne/styled/optionPicker.js

```diff
@@ -0,0 +1,94 @@
+import styled from 'styled-components';
+import File from '../../icons/file';
+import attachThemeAttrs from '../../../../styles/helpers/attachThemeAttrs';
+import RadioGroup from '../../../RadioGroup';
+import MergeFiles from '../../icons/mergeFiles';
+
+export const OptionContainer = attachThemeAttrs(styled.li)`
```

Comment:
you can use li elements to define each item but for this you need to wrap it in ol or ul element, look here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/li
`It must be contained in a parent element: an ordered list <ol>, an unordered list <ul>`

Here there is an example with all the accessibility specs where you can use ul and li for the radio buttons combined with the attribute role: https://www.w3.org/WAI/ARIA/apg/patterns/radio/examples/radio-activedescendant/

Another option is to simply use div everywhere with the use of role like this: https://www.w3.org/WAI/ARIA/apg/patterns/radio/examples/radio/

But since we are using input and labels I think the easiest solution is to simply change li for a span and keep the picker like this one: https://www.lightningdesignsystem.com/components/visual-picker/